girder/api/v1/resource.py

```diff
@@ -180,7 +180,15 @@ def _lookUpToken(self, token, parentType, parent):
         raise RestException('Child resource not found: %s(%s)->%s' % (
             parentType, parent.get('name', parent.get('_id')), token))
 
-    def _lookUpPath(self, path, test, user):
+    def _lookUpPath(self, path, user, test=False):
```

Comment:
You'll need to change the order below when you call this, as the param order changed

    return self._lookUpPath(params['path'], self.getCurrentUser(), test)